region/norfair/west/Crocomire Speedway.json

```diff
@@ -592,6 +886,47 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [4, 5],
+      "name": "Come In Shinecharging, Leave With Spark",
+      "entranceCondition": {
+        "comeInShinecharging": {
+          "length": 1,
```

Comment:
- Using the tile for runway makes storing the shinespark extremely tight.  
- Sliding into the door and off the ledge is a slower fall so not really easier overall
- stopping before the door means you work with fewer frames but the initial platforming is easier.

They were all about equivalent, but the trick to stopping in room took some tries to learn.

Reply:
I think its a little more difficult than other extreme level shinecharge movement strats.